src/Test/AbstractSchemaResetter.php

```diff
@@ -24,4 +24,15 @@ protected function runCommand(Application $application, string $command, array $
             throw new \RuntimeException(\sprintf('Error running "%s": %s', $command, $output->fetch()));
         }
     }
+
+    /**
+     * @param list<string> $availableObjectsToReset
+     * @param list<string> $candidateObjectsToReset
+     */
+    protected static function validateObjectsToReset(string $objectsType, array $availableObjectsToReset, array $candidateObjectsToReset): void
```

Comment:
this check is made at runtime because it would require foundry bundle to read doctrine's configuration through a compiler pass, which is a little bit weird. WDYT?

Reply:
I'm fine with runtime.